File: swap.cpp
#include<iostream>

int main(){
    int a,b;
    int temp;
    std::cout << "Enter a and b\n";
    
    std::cin >> a;
    std::cin >> b;

    std::cout << "a is : "<< a<<"\n";
    std::cout << "b is : "<< b<<"\n";

    //to swap the numbers
    temp = a;
    a = b;
    b = temp;

    std::cout << "Updated a is : "<< a<<"\n";
    std::cout << "Updated b is : "<< b<<"\n";

    return 0;
    
}

File: swap.java
package series;
import java.util.Scanner;

public class swap {

	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);
		
		int a,b;
		
		System.out.println("Enter the first number(a) : ");
		a = sc.nextInt();
		
		System.out.println("Enter the second number(b) : ");
		b =sc.nextInt();
		
		int temp;
		
		temp = a;
		a = b;
		b = temp;
		
		System.out.println("Updated a is : "+a);
		System.out.println("Updated b is : "+b);

	}

}
